diff --git a/dist/version-request-l6IJdFM6.js b/dist/version-request-l6IJdFM6.js
index 3709ecce7d9241a627695915522858f2e183d29d..c3b07f315ca505fc5737181eeaca8e6ef743adc3 100644
--- a/dist/version-request-l6IJdFM6.js
+++ b/dist/version-request-l6IJdFM6.js
@@ -151,7 +151,7 @@ function onVersionRequest(provider) {
 				packages: Array.from(updatedPackages.values())
 			});
 			if (requests.length === 0) return;
-			await createVersionCommit(this.cwd, "Version Packages");
+			await createVersionCommit(this.cwd, "chore: version packages");
 			let baseLock;
 			let parent;
 			const tasks = [];
